### ISSUE TITLE
material-design-icons failing to install with "ERR! cb() never called!"

### ISSUE THREAD

**Issue by shyndman** [npm/npm#11976]
Hi there,

I'm a maintainer of the [google/material-design-icons](https://github.com/google/material-design-icons) repo. We've received a report that our 2.2.2 release fails to install with `"ERR! cb() never called!"` ([issue](https://github.com/google/material-design-icons/issues/342)) . I've been able reproduce myself, with the two versions npm 3.7.3 and 3.8.2. Version 2.2.1 works fine.

I'd assumed that I failed to publish properly, so went ahead and cut a new 2.2.3 version, but I'm still seeing the same problem.

Any sense of what's going on? Nothing stands out when I run `npm -d install`.

The only thing I did notice is that package.json downloaded for 2.2.1 is quite different than 2.2.2's.

**Comment by pYr0x** [comment]
hi,
same problem with https://github.com/canjs/canjs
using:
* npm 3.7.3
* node 5.9.0

**Comment by only-style** [comment]
how to solve the problem guys?

- nodejs - 5.9.0
- npm - 3.7.3

**Comment by erykpiast** [comment]
I got the same, versions like @pYr0x has. But cannot say what package exactly, it just fails when I try to install dependencies for my app.

**Comment by Klathmon** [comment]
@erykpiast I figured out which package it was by doing the old fashioned "delete half my deps and try again, if it works, switch which half i removed, repeat until i'm down to 1 package" 

In my case it ended up being the google/material-design-icons package but only versions after 2.2.1

**Comment by erykpiast** [comment]
So, in my case it's `typescript 0.9.1-1` (as a `devDependency`).

**Comment by fzaninotto** [comment]
Same problem with `npm install ng-admin@1.0.0-alpha1`

**Comment by arman-mukatov** [comment]
That should work:

`npm cache clean`

simply remove the .npm folder from our home directory.

`rm -rf ~/.npm`

and try again. It worked for me!

**Comment by xorgy** [comment]
I'm also seeing this issue.

**Comment by unindented** [comment]
node 5.9.0
npm 3.7.3

TravisCI is alerting me of this issue across a bunch of projects, and I'm seeing it locally too.

`npm cache clean` didn't fix it for me.

**Comment by greyhwndz** [comment]
have the same error on my end:
`npm ERR! cb() never called!`
node: v5.9.0
npm: 3.8.2

Tried `npm cache clean` & `rm -rf ~/.npm` to no effect

**Comment by pornel** [comment]
```sh
npm cache clean
npm init foo
npm i -S yr
npm ERR! cb() never called!
```

Node v5.9.1 npm 3.8.5 (OS X 10.11.4)

**Comment by pYr0x** [comment]
@shyndman just a new release on npm worked for your repo... Maybe you test this with yours

**Comment by erykpiast** [comment]
Hey, does anybody know some workaround for this for Windows? I've already tried to remove `%HOME%\AppData\Roaming\npm-cache` and `%HOME%\AppData\Local\Temp`. No results, `npm install typescript@0.9.1-1` still finishes with error.

**Comment by unindented** [comment]
@erykpiast If @shyndman has no new releases planned, your best option may be to use Node 4.4.x or 5.8.x. Not sure why only 5.9.x is affected.

**Comment by erykpiast** [comment]
@unindented, ok, thank you!

**Comment by xorgy** [comment]
I may not get around to running a bisect to find the commit for this this week; if somebody else could then we'd have a better idea what's going on. @unindented believes that this does not happen in the 5.8 series, so it won't be a painful bisect.

**Comment by shyndman** [comment]
@unindented, @pYr0x When this problem first cropped up, I cut a new release. It had didn't have any effect.

**Comment by unindented** [comment]
@shyndman 👍

Then our best hope is to bisect. I'd do it if I knew how to check out and use specific commits of the npm project...

**Comment by sasidhar** [comment]
I am also facing the `cb() never called!` issue.

My environment
Windows 7 64 Bit
NPM : 3.7.3
Node : 5.9.1

When I tried `npm cache clean` and tried `npm install`, it is failing while installing different packages.

Can't do much with this error on Windows. :(

**Comment by sasidhar** [comment]
Reverted to 

NPM : 2.14.20 and
Node : 4.4.1
on Windows 7 64 Bit

`npm install` is working fine without any issues.

**Comment by asottile** [comment]
fwiw, this seems fixed (magically?) for me.

npm: 3.8.3
node 5.10.0
Linux asottile-VirtualBox 3.19.0-56-generic #62-Ubuntu SMP Thu Mar 10 22:40:07 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux

**Comment by darkwinternight** [comment]
Yup, magically fixed itself today.

npm: 3.8.5
node: 5.10.0
Mac OS X 10.11.4

**Comment by LukasMa** [comment]
Fails on:
Windows 10 64bit
Node: 5.91
NPM: 3.7.3

Will update to latest Node and NPM and try again.

**Comment by xorgy** [comment]
Noting also that this seems to be not happening as of Node 5.10.

**Comment by sscaff1** [comment]
I was having this issue too and it magically resolved when I upgrade to node 5.11.
